programs/mango-v3-reimbursement/src/instructions/create_vault.rs

```diff
@@ -31,7 +32,7 @@ pub struct CreateVault<'info> {
         seeds = [b"Mint".as_ref(), group.key().as_ref(), &token_index.to_le_bytes()],
         bump,
         mint::authority = group,
-        mint::decimals = mint_decimals,
+        mint::decimals = mint.decimals,
```

Comment:
For simplicity we could also create the claim target token account at the same time here, otherwise it'll need to be created externally

Reply:
Hmm I tried this, and I am getting a compilation error, can you take a look, unsure if we can create a claim mint ata in the same ix, where we create the claim mint, I think I faced this yday, hence I had removed it
![image](https://user-images.githubusercontent.com/89031858/196395685-8cb98274-3a12-4545-9da6-bfb7b41c46da.png)

Reply:
The issue seems to be not the mint, but that initing an ATA needs the authority AccountInfo, not just the Pubkey. I'll look at it a bit.

Reply:
Yeah, looks like that's a limitation of the ATA program. Solution: pass in the `claim_transfer_destination` account along with a `has_one` on the group for it.

Reply:
Btw, I don't think we need init-if-needed, since the new mint is guaranteed to not have an ata yet